test/integration/driver.js

```diff
@@ -143,7 +143,7 @@ export class WebExtensionDriver {
     webext.id = await driver.installAddon(webext.extensionPath);
     webext.hostname = await driver.executeScript(`
       const Cu = Components.utils;
-      const Services = ChromeUtils.import("resource://gre/modules/Services.jsm");
+      ChromeUtils.defineModuleGetter(this, "Services", "resource://gre/modules/Services.jsm");
```

Comment:
FWIW, this gets integration tests running for me on windows, too!

<img width="960" alt="capture" src="https://user-images.githubusercontent.com/21687/50020555-4f013a00-ffa4-11e8-98b1-0296f958d766.PNG">